pages/accountLists/[accountListId]/contacts/ContactsWrapper.tsx

```diff
@@ -77,6 +77,15 @@ export const ContactsWrapper: React.FC<Props> = ({
   );
   const [filterPanelOpen, setFilterPanelOpen] = useState(true);
 
+  // Only allow the ids filter in map view, and remove the ids filter in other views
+  const activeFilters = useMemo(() => {
+    const { ids, ...otherFilters } = activeFiltersRaw;
+    if (viewMode === TableViewModeEnum.Map) {
+      return ids?.length ? { ids } : {};
+    }
+    return otherFilters;
+  }, [viewMode, activeFiltersRaw]);
```

Comment:
Does `ids` return an empty array or undefined? as you could write it as the below and save the extra If statement.
```suggestion
    const { ids, ...otherFilters } = activeFiltersRaw;
    if (viewMode === TableViewModeEnum.Map && ids?.length) {
      return { ids }
    }
    return otherFilters;
  }, [viewMode, activeFiltersRaw]);
```